ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py

```diff
@@ -672,15 +672,17 @@ def test_tensorflow_batch_normalization(
         min_value=0,
         shape=(3, 5),
     ),
-    prob=helpers.array_values(dtype=ivy.float16, shape=(3, 5), min_value=0),
-    scale=helpers.array_values(dtype=ivy.float16, shape=(3, 5), min_value=0),
+    prob=helpers.floats(min_value=0, max_value=1),
+    scale=st.booleans(),
+    noise_shape=helpers.array_values(dtype=ivy.float16, shape=(3, 5), min_value=0)
```

Comment:
You're not supposed to generate array values here, you have to generate a broadcastable shape here.
You can use `helpers.mutually_broadcastable_shapes`  for that